selfdrive/camerad/cameras/camera_qcom2.cc

```diff
@@ -681,6 +681,18 @@ void CameraState::camera_open() {
   LOG("-- Configuring sensor");
   if (camera_id == CAMERA_ID_AR0231) {
     sensors_i2c(init_array_ar0231, std::size(init_array_ar0231), CAM_SENSOR_PACKET_OPCODE_SENSOR_CONFIG, true);
+
+    if (ci.stats_offset != -1 || ci.registers_offset != -1) {
+      uint16_t smia_test = 0x1802;
+      if (ci.stats_offset != -1) smia_test |= (1 << 7);
+      if (ci.registers_offset != -1) smia_test |= (1 << 8);
+
+      struct i2c_random_wr_payload smia_test_reg[] = {
+        {0x3064, smia_test},
+      };
+      sensors_i2c(smia_test_reg, std::size(smia_test_reg), CAM_SENSOR_PACKET_OPCODE_SENSOR_CONFIG, true);
```

Comment:
Should be safe to always send it, but gating behind the env variable to ensure everything is the same as before merging https://github.com/commaai/openpilot/pull/24353